src/together/error.py

```diff
@@ -23,9 +23,12 @@ def __init__(
             if isinstance(message, TogetherErrorResponse)
             else message
         )
-        self._message = f"Error code: {http_status} - {_message}"
+        if http_status is not None:
+            self._message = f"Error code: {http_status} - {_message}"
+        else:
+            self._message = str(_message)
```

Comment:
If there is no http_status, the base exception starts with "None - …", which is confusing and unnecessary if the error is not related to an HTTP request